### DISCUSSION TITLE
Supporting optix recipe

### DISCUSSION THREAD

**Discussion by bchoineubility** [OE4T/meta-tegra#1517] [Q&A]
Hi,

Does meta-tegra support nvidia optix also?
BR,
Mark

**Comment by madisongh** [comment]
As far as I know, NVIDIA does not support Optix on the Jetson platforms.

**Comment by ichergui** [comment]
I agree @madisongh. I'm not aware about that specifically on the Jetson platforms. 
@bchoineubility Could you please elaborate a bit about your project ?

**Reply by bchoineubility** [reply]
I am building ros2-humble with yocto. 
A package check dependency for optix, but, it was not required option. 

So, I googled for optix and found this.
[nvida optix tool](https://developer.nvidia.com/designworks/optix/download)

BUT, I am not sure it was what ros package depend.

**Reply by ichergui** [reply]
Please check the NOTE in the link you shared.  

**NOTE:** Requires NVIDIA R535 driver or newer. You may need a [Beta Driver](https://www.nvidia.com/Download/Find.aspx?lang=en-us) for certain operating systems.

If I'm not mistaken the optix requires dGPU hence nvidia driver R535 is required. and not iGPU (Jetson platforms).
